ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusTextMessage.qml

```diff
@@ -12,6 +12,7 @@ Item {
     property StatusMessageDetails messageDetails: StatusMessageDetails {}
     property bool isEdited: false
     property bool convertToSingleLine: false
+    property bool stripeHtmlTags: false
```

Comment:
Typo, `strip`

Reply:
Updated this in all places as part of this PR